tests/sentry/backup/test_comparators.py

```diff
@@ -396,6 +400,211 @@ def test_good_hash_obfuscating_comparator_scrubbed():
     ]
 
 
+DEPENDENCIES = dependencies()
+
+
+def test_good_foreign_key_comparator():
+    cmp = ForeignKeyComparator(
+        {k: v.model for k, v in DEPENDENCIES["sentry.UserEmail"].foreign_keys.items()}
```

Comment:
are the caps in `sentry.UserEmail` done on purpose for testing?

Reply:
The dependency sorting function returns models named in camelCase, whereas the exported JSON format uses all lowercase. I decided to keep things all lowercase for the JSON format to preserve compatibility, but imo this was a mistake that we should avoid propagating further in new code. This is because it is possible (in the general case) to deduce what the words are in a camelCased (or snake_cased, or kebab-cased...) name `sentry.UserEmail` (ex, so that I can convert it to `get_user_email()` for a standardized method call), whereas `sentry.useremail` is one giant unrecoverable blob.